### PR TITLE
[NASA] Add outdoor temperature sensor

### PR DESCRIPTION
**Pull request by Mongey**
I'm not sure what this should be called, but this is what it represents on mine.

### CODE REVIEW

**Comment by atanasenko** [comment]
@Mongey @lanwin  Can you have a look at #94? Maybe that would be easier than to try and add all the different variants.
Those sensors are almost all the same, but different devices might use different message numbers.
As a followup we could introduce some shorthand on configuration level for known sensors, and not have it be coded into protocol code.

**Comment by atanasenko** [comment]
Also, this:
image

It's currently around -4 outside, this has to be accounted for.

**Comment by Mongey** [comment]
Yeah #94 might be a better solution than adding all the specific ones I have found for my system

**Comment by atanasenko** [comment]
I'll update it later today, I think it should be easy to define sensors just using python file, as it's basically a bunch of sensor config options and a message number.

**Comment by lanwin** [comment]
I would think that its a good idea to add the basic stuff as specific sensor, since that makes it a lot easier for non experienced uses. Understand the nasa serial numbers and type is not that easy for someone with no developer background and only want to add its AC to Homeassist.

The custom component approche works for NASA but not for NonNASA.

File: components/samsung_ac/protocol_nasa.cpp [hunk 1]
```diff
@@ -699,6 +699,14 @@ namespace esphome
                     ESP_LOGW(TAG, "s:%s d:%s NASA_OUTDOOR_CONTROL_WATTMETER_TOTAL_SUM_ACCUM %f", source.c_str(), dest.c_str(), value);
                     return;
                 }
+                if ((uint16_t)message.messageNumber == 0x8204)
```
Comment (lanwin) [inline components/samsung_ac/protocol_nasa.cpp:702]: Can you please add that as VAR_OUT_SENSOR_AIROUT to the enum.

**Review by lanwin — CHANGES_REQUESTED** [review]
Thank you for that pullrequest. I added a little change request.

**Comment by lanwin** [comment]
> Yeah #94 might be a better solution than adding all the specific ones I have found for my system

Please dont do that. This approach did not work for nasa devices.

**Pull request merged**